### DISCUSSION TITLE
Texstudio refresh

### DISCUSSION THREAD

**Discussion by italo963** [texstudio-org/texstudio#2342] [Q&A] [answered]
Hi you all.
    My texstudio application is corrupted somehow.
I just want to start fresh on a new instance of it.
it.  It was working well for a long long time.   But now it won't 
go to the starting page and is generally acting balmy.

After wasting so much time trying to get it to work right
I would just like to refresh or reinstall to factory settings.
I tried uninstalling and reinstalling but it still doesn't work.

Can anyone help ?  Thanks in advance !

**Comment by sunderme** [comment, marked as answer]
options/restore default settings

**Reply by italo963** [reply]
Hey thanks,  I think that worked.
Thanks a million !  A good experience on github.
Hope it's not just beginners luck haha !
